facilitator/src/logging.rs

```diff
@@ -0,0 +1,59 @@
+use env_logger::fmt::Formatter;
+use log::{Level, Record};
+use serde::Serialize;
+use std::io::Write;
+
+/// Severity maps `log::Level` to Google Cloud Platform's notion of Severity.
+/// https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#LogSeverity
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "UPPERCASE")]
```

Comment:
The point of using `rename_all = "UPPERCASE"` is that you can then have the variants be named in the more conventional Rust fashion, which would be `Debug, Info, Warning, Error`, but still have them be serialized in uppercase.

https://serde.rs/container-attrs.html

Reply:
Oh that's an embarrassing misunderstanding! Thanks will fix this.